.github/workflows/java-continuous-integration.yml

```diff
@@ -0,0 +1,81 @@
+# Description
+# ===========
+# This workflow is triggered each time
+# commits are pushed to GitHub or a pull request is opened.
+# It launches three jobs in parallel : a build with java 8,
+# a build with java 11 and a SonarCloud analysis.
+---
+name: Java CI
+
+on: [push, pull_request]
```

Comment:
I don't know if you saw it but there is also a file to handle automatic release + draft release notes.
Maybe we could add it too, don't you think ?

Reply:
I can add this functionality too, I can check what is already done. 
It's used by cnes-report, no ?

Reply:
yes it is